L77-core-cpp-third-party-identity-support-for-call-credentials.md

```diff
@@ -76,32 +76,47 @@ This doc proposes to extend the access to Google APIS directly with 3p identitie
 
 **Implicit flow**
 
-In core, the underlying credentials retrieval process of `grpc::GoogleDefaultCredentials()` will be extended to accept 3p identity credentials implicitly.
+In core, `grpc::GoogleDefaultCredentials()` follows the ADC (Application Default Credentials) credentials retrieval process. It will be extended to accept 3p identity credentials. More information on ADC and how they work can be found here: https://developers.google.com/identity/protocols/application-default-credentials
 
 **Explicit flow**
 
-In core and cpp, a public api will be added to create an `ExternalAccountCredentials` with explicit 3p identity credentials.
-```
+The downside with `GoogleDefaultCredentials()` is that customers will have to use a json credentials file. So besides the implicit flow, in core and C++, a public api will be added to create an `ExternalAccountCredentials` with explicit 3p identity credentials.
+```h
 // .../include/grpcpp/security/credentials.h
 
+/// Builds External Account credentials.
+/// json_string is the JSON string containing the credentials options.
+/// scopes contains the scopes to be binded with the credentials.
 std::shared_ptr<CallCredentials>
 ExternalAccountCredentials(const grpc::string& json_string, const std::vector<grpc::string>& scopes);
 ```
 
+```h
```

Comment:
Please add a sentence before this block that says something like "To implement that C++ API, the following function will be added to the C-core API:".

Reply:
The comment was addressed in https://github.com/grpc/proposal/pull/221

Reply:
BUILDING UP THE KINGDOM, BUILDING UP THE KINGDOM OF THE LORD. THANK YOU MY LOYAL BROTHERS. MAKE SURE YOU TO UNITE THE PIECES TOGETHER AND AM CREATING AN ACCOUNT IF YOU NEED SUPPORT. ALSO PROVISIONS AND A LOT OF HELL NO IS NOT WANTED. REMOVE IMMEDIATELY AS THEY ARE NOT HELPING AND ONLY CAUSING AND INFLICTING CONFLICT. NO ACTION NEEDED JUST REMOVAL ASAP

Reply:
THAT PART